aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -858,6 +882,35 @@ module aptos_framework::fungible_asset {
         deposit_with_ref(transfer_ref, to, fa);
     }
 
+    /// Mutate specified fields of the fungible asset's `Metadata`.
+    public fun mutate_metadata(
+        metadata_ref: &MutateMetadataRef,
+        name: Option<String>,
```

Comment:
I don't think we need these to be Option < >. Callers can pass old values in for values that don't change

Reply:
Thinking from a DevX perspective, it seems a bit odd to be forced to retrieve the old metadata and redefine the same information in `mutate_metadata`. Though, having to specify `Option::none()` isn't much different...

Let me ask a few others for their opinions, as well.